app/models/marc_fields/bound_with_note_for_access_panel.rb

```diff
@@ -0,0 +1,26 @@
+###
+#  MarcBoundWith class to return 590 fields that have a $c
+###
+class BoundWithNoteForAccessPanel < BoundWithNote
+  def values
+    extracted_fields.map do |field, subfields|
+      id = subfields.find { |subfield| subfield.code == 'c' }.value[/^(\d+)/]
```

Comment:
And `id` here is a catkey like above? If so, would you consider renaming the variable to `ckey` (also, like above)?

Reply:
Yes, good point: changing the hash here would break stuff that depends on it. NBD.